full/src/test/java/apoc/config/ConfigTest.java

```diff
@@ -48,20 +56,28 @@ public class ConfigTest {
             entry(APOC_TTL_SCHEDULE, "PT1M")
     );
 
-    @Rule
-    public final ProvideSystemProperty systemPropertyRule 
-            = new ProvideSystemProperty("foo", "bar")
-            .and("apoc.import.enabled", "true")
-            .and("apoc.trigger.refresh", "2000")
-            .and("apoc.jobs.scheduled.num_threads", "4")
-            .and("apoc.static.test", "one")
-            .and("apoc.jdbc.cassandra_songs.url", "jdbc:cassandra://localhost:9042/playlist");
-
     @Rule
     public DbmsRule db = new ImpermanentDbmsRule();
 
+    @BeforeClass
+    public static void setUp() throws Exception {
+        final File conf = new File(directory, "apoc.conf");
+        try (FileWriter writer = new FileWriter(conf)) {
+            writer.write(String.join("\n",
+                    "foo=bar",
+                    "apoc.import.enabled" + "=true",
+                    TRIGGER_REFRESH + "=2000",
+                    APOC_CONFIG_JOBS_SCHEDULED_NUM_THREADS + "=4",
+                    "apoc.static.test=one",
+                    "apoc.jdbc.cassandra_songs.url=jdbc:cassandra://localhost:9042/playlist"
+            ));
+        }
+
+        System.setProperty(SUN_JAVA_COMMAND, "config-dir=" + directory.getAbsolutePath());
+    }
+
```

Comment:
My initial idea was to give a shot using an "apoc.conf" like TriggerNewProceduresTest, which seems not to be flaky.

But rethinking about it, perhaps it is the latter that causes the flakyness, 
since [this line](https://github.com/neo4j-contrib/neo4j-apoc-procedures/blob/4.4/core/src/test/java/apoc/trigger/TriggerNewProceduresTest.java#L88) likely remains during the ConfigTest, hence the error.

Now I did the vice versa, ie changed all the tests that leverage the "apoc.conf" file consistently with the `ConfigTest` one.